modules/ROOT/pages/amazon/amazon-dynamodb-connector-examples.adoc

```diff
@@ -0,0 +1,700 @@
+            },
+            {
+                "IndexName": "TitleIndex",
+                "Projection": {
+                    "ProjectionType": "ALL"
+                },
+                "ProvisionedThroughput": {
+                    "WriteCapacityUnits": 5,
+                    "ReadCapacityUnits": 5
+                },
+                "KeySchema": [
+                    {
+                        "KeyType": "HASH",
+                        "AttributeName": "Title"
+                    },
+                    {
+                    	"KeyType": "RANGE",
+                    	"AttributeName": "IssueId"
+                    }
+                ]
+            } as Object {
+            	class: "org.mule.extension.dynamodb.api.model.GlobalSecondaryIndex"
+            },
+            {
+                "IndexName": "DueDateIndex",
+                "Projection": {
+                    "ProjectionType": "ALL"
+                },
+                "ProvisionedThroughput": {
+                    "WriteCapacityUnits": 5,
+                    "ReadCapacityUnits": 5
+                },
+                "KeySchema": [
+                    {
+                        "KeyType": "HASH",
+                        "AttributeName": "DueDate"
+                    }
+                ]
+            } as Object {
+            	class: "org.mule.extension.dynamodb.api.model.GlobalSecondaryIndex"
+            }
+        ]]' doc:name="Secondary" variableName="secondary"/>
+
+		<dynamodb:create-table doc:name="Create table" config-ref="Amazon_DynamoDB_Configuration" tableName="Issues" readCapacityUnits="5" writeCapacityUnits="5" globalSecondaryIndexes="#[vars.secondary]">
+		<dynamodb:attribute-definitions>
+			<dynamodb:attribute-definition attributeName="IssueId" attributeType="STRING" />
+			<dynamodb:attribute-definition attributeName="Title" attributeType="STRING" />
+			<dynamodb:attribute-definition attributeName="CreateDate" attributeType="STRING" />
+			<dynamodb:attribute-definition attributeName="DueDate" attributeType="STRING" />
+		</dynamodb:attribute-definitions>
+		<dynamodb:key-schemas>
+			<dynamodb:key-schema-element attributeName="IssueId" keyType="HASH" />
+			<dynamodb:key-schema-element attributeName="Title" keyType="RANGE" />
+		</dynamodb:key-schemas>
+	</dynamodb:create-table>
+		<logger level="INFO" doc:name="Logger"
+    message="#[payload]"/>
+</flow>
+</mule>
+----
+
+=== Run the Project
+
+Save and run the project as a Mule app:
+
+. In *Package Explorer*, right-click the project and click *Run As* > *Mule Application*.
+. Open a browser and check the response after you enter the `+http://localhost:8081/createtable+` URL.
++
+You should see the generated response from the `Create table` operation in the Mule console.
+
+[[delete-table]]
+== Delete an Amazon DynamoDB Table
+
+This example shows how to use Amazon DynamoDB connector to delete an Amazon DynamoDB table.
```

Comment:
Anything to say about under what circumstances our audience wants or needs to do this?